test/widgets/content_test.dart

```diff
@@ -290,6 +297,79 @@ void main() {
 
   testContentSmoke(ContentExample.mathBlock);
 
+  testContentSmoke(ContentExample.strong);
+
+  testContentSmoke(ContentExample.emphasis);
+
+  group('InlineCodeNode', () {
+    testContentSmoke(ContentExample.inlineCode);
+
+    testWidgets('maintains font-size ratio with surrounding text', (tester) async {
+      /// Simulate a nested span's style by merging all ancestor-span styles,
+      /// starting from the root.
+      ///
+      /// This isn't how the style actually gets applied in the code under test,
+      /// but it should hopefully simulate it closely enough.
```

Comment:
```suggestion
      /// This isn't how the style actually gets applied in the code under test
      /// (because that happens inside the engine, in SkParagraph),
      /// but it should hopefully simulate it closely enough.
```

That helps answer the question of why we *don't* just do what the code under test actually does.